webhook-validation-proxy.go

```diff
@@ -44,7 +44,7 @@ func validateAndProxy(w http.ResponseWriter, req *http.Request) {
 		log.Printf("%s, err: %s", msg, err)
 		w.WriteHeader(http.StatusBadRequest)
 		w.Write([]byte(msg))
+	} else {
```

Comment:
@czerwinb thanks for noticing it, broken with https://github.com/mslusarczyk/webhook-validation-proxy/commit/175c98e8458fd95d8342531cc2d614ca6611e651. 'Retrun' in if is more idiomatic golang. Please change it to 'return'.